src/test/compile-fail/E0501.rs

```diff
@@ -26,10 +26,12 @@ fn foo(a: &mut i32) {
         inside_closure(a)
     };
     outside_closure_1(a); //[ast]~ ERROR cannot borrow `*a` as mutable because previous closure requires unique access
-                         //[mir]~^ ERROR cannot borrow `*a` as mutable because previous closure requires unique access
```

Comment:
Of course, there was a reason that these comments were indented so much: To allow ease of comparison with the line above.

Having said that, we all hope that compare-mode=nll will provide a much better path forward, so I don't really care to try to preserve details like this here.